client/src/helpers/APICalls/bookingService.ts

```diff
@@ -1,34 +1,51 @@
 import { BookingStatusType } from '../../context/useRequestContext';
 import { BookingApiData, BookingRequest } from '../../interface/BookingApiData';
-import { bookings } from '../../mocks/mockBookings';
+import { FetchOptions } from '../../interface/FetchOptions';
 
-export const getBookings = (): Promise<Array<BookingRequest>> => {
-  return Promise.resolve(bookings);
+interface UpdateBooking {
```

Comment:
In the future please refrain from making changes in other features as the booking helper functions are not a part of this PR. You can create a small PR at any time to refactor existing code :)